public/css/docs.css

```diff
@@ -1269,3 +1247,542 @@ footer .container {
 
 
 /*-------------------------/.MARGIN/PADDING-------------------------*/
+
+html {
+    font-size: 100%;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%
+}
+
+body {
+    margin: 0;
+    padding: 0
+}
+
+pre,
+code,
+address,
+caption,
+th,
+figcaption {
+    font-size: 1em;
+    font-weight: normal;
+    font-style: normal
+}
+
+fieldset,
+iframe,
+img {
+    border: none
+}
+
+caption,
+th {
+    text-align: left
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0
+}
+
+article,
+aside,
+footer,
+header,
+nav,
+main,
+section,
+summary,
+details,
+hgroup,
+figure,
+figcaption {
+    display: block
+}
+
+audio,
+canvas,
+video,
+progress {
+    display: inline-block;
+    vertical-align: baseline
+}
+
+button {
+    font: inherit;
+    vertical-align: middle
+}
+
+*,
+*:before,
+*:after {
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%
+}
+
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, Verdana, sans-serif;
+    font-size: 18px;
+    line-height: 1.5;
+    color: #333;
+    font-weight: 300
```

Comment:
I am having trouble with my 👁 comfort with this font at weight 300. It's a little too thin at `#333`.
I would be happy exploring any of these options.
- new font
- color that approaches `#111`
- font weight `normal`

Personally, I like this font and it a little darker looks nice on my Mac but I will pull my windows machine too and double check it.